static/app/components/replays/breadcrumbs/breadcrumbItem.tsx

```diff
@@ -63,6 +66,91 @@ function BreadcrumbItem({
 
   const forceSpan = 'category' in frame && FRAMES_WITH_BUTTONS.includes(frame.category);
 
+  const renderDescription = useCallback(() => {
+    return typeof description === 'string' ||
+      (description !== undefined && isValidElement(description)) ? (
+      <Description title={description} showOnlyOnOverflow isHoverable>
+        {description}
+      </Description>
+    ) : (
+      <InspectorWrapper>
+        <ObjectInspector
+          data={description}
+          expandPaths={expandPaths}
+          onExpand={onInspectorExpanded}
+          theme={{
+            TREENODE_FONT_SIZE: '0.7rem',
+            ARROW_FONT_SIZE: '0.5rem',
+          }}
+        />
+      </InspectorWrapper>
+    );
+  }, [description, expandPaths, onInspectorExpanded]);
+
+  const renderComparisonButton = useCallback(() => {
+    return frame?.data && 'mutations' in frame.data ? (
+      <div>
+        <OpenReplayComparisonButton
+          replay={replay}
+          leftTimestamp={frame.offsetMs}
+          rightTimestamp={
+            (frame.data.mutations.next?.timestamp ?? 0) -
+            (replay?.getReplay().started_at.getTime() ?? 0)
+          }
+          size="xs"
+        >
+          {t('Open Hydration Diff')}
+        </OpenReplayComparisonButton>
+      </div>
+    ) : null;
+  }, [frame?.data, frame.offsetMs, replay]);
+
+  const renderCodeSnippet = useCallback(() => {
+    return extraction?.html ? (
+      <CodeContainer>
+        <CodeSnippet language="html" hideCopyButton>
+          {beautify.html(extraction?.html, {indent_size: 2})}
+        </CodeSnippet>
+      </CodeContainer>
+    ) : null;
+  }, [extraction?.html]);
+
+  const renderIssueLink = useCallback(() => {
+    return isErrorFrame(frame) || isFeedbackFrame(frame) ? (
+      <CrumbErrorIssue frame={frame} />
+    ) : null;
+  }, [frame]);
+
+  const hasNewTimelineUI = useHasNewTimelineUI();
+  const timeString = new Date(frame.timestampMs).toISOString();
+  const startTimeString = new Date(startTimestampMs).toISOString();
```

Comment:
there might be `frame.timestamp` already that is a Date type

Reply:
It looks like `timestamp` doesn't exist on `SpanFrame` so but `timestampMs` exists on all the types of `ReplayFrame`s so I might have to leave it as is.